### PR TITLE
Correctly trigger Actuator Target Events

### PR DESCRIPTION
**Pull request by wba2hi**
Closes: #29

### CODE REVIEW

File: kuksa-sdk/src/main/kotlin/org/eclipse/kuksa/DataBrokerTransporter.kt [hunk 1]
```diff
@@ -176,4 +172,30 @@ internal class DataBrokerTransporter(
 
         return subscription
     }
+
+    private fun createEntryUpdate(
```
Comment (Chrylo) [inline kuksa-sdk/src/main/kotlin/org/eclipse/kuksa/DataBrokerTransporter.kt:176]: How about an extension here:

```
fun Types.DataEntry.Builder.setDatapoint(field: Field, datapoint: Types.Datapoint): Types.DataEntry.Builder {
    when (field) {
        Field.FIELD_ACTUATOR_TARGET -> {
            this.actuatorTarget = datapoint
        }

        else -> {
            this.value = datapoint
        }
    }

    return this
}
```

```
val builder = Types.DataEntry.newBuilder()
    .setDatapoint(field, updatedDatapoint)
    .setPath(vssPath)
    .build()

return KuksaValV1.EntryUpdate.newBuilder()
    .setEntry(builder)
    .addFields(field)
    .build()
   ```

File: kuksa-sdk/src/main/kotlin/org/eclipse/kuksa/DataBrokerConnection.kt [hunk 2]
```diff
@@ -204,18 +205,22 @@ class DataBrokerConnection internal constructor(
...
      * Compared to [update] with only one [Property] and [Datapoint], here multiple [SetResponse] will be returned
      * because a [VssSpecification] may consists of multiple values which may need to be updated.
      *
      * @throws DataBrokerException in case the connection to the DataBroker is no longer active
      * @throws IllegalArgumentException if the [VssProperty] could not be converted to a [Datapoint].
      */
-    suspend fun update(vssSpecification: VssSpecification): List<SetResponse> {
+    suspend fun update(
+        vssSpecification: VssSpecification,
+        fields: List<Field> = listOf(Field.FIELD_VALUE),
```
Comment (Chrylo) [inline kuksa-sdk/src/main/kotlin/org/eclipse/kuksa/DataBrokerConnection.kt:218]: Can we use Collection here? :)

**Review by Chrylo — CHANGES_REQUESTED** [review]

**Comment by Chrylo** [comment]
Tested setting the Vehicle.Cabin.HVAC.AmbientAirTemperature via the new Field type. Works in combination with the mock client.

LGTM after the code findings are fixed.

File: kuksa-sdk/src/main/kotlin/org/eclipse/kuksa/DataBrokerTransporter.kt (hunk 1 above)
Comment (wba2hi) [reply kuksa-sdk/src/main/kotlin/org/eclipse/kuksa/DataBrokerTransporter.kt]: Done

File: kuksa-sdk/src/main/kotlin/org/eclipse/kuksa/DataBrokerConnection.kt (hunk 2 above)
Comment (wba2hi) [reply kuksa-sdk/src/main/kotlin/org/eclipse/kuksa/DataBrokerConnection.kt:218]: Yes of course. Done

**Review by Chrylo — APPROVED** [review]

**Review by Chrylo — APPROVED** [review]

**Pull request merged**